.github/workflows/docker.yml

```diff
@@ -57,15 +59,25 @@ jobs:
           - sytest_image_tag: focal
             dockerfile: synapse
             tags: "matrixdotorg/sytest-synapse:focal"
+            build_arg: "SYTEST_IMAGE_TAG=focal"
           - sytest_image_tag: buster
             dockerfile: synapse
             tags: "matrixdotorg/sytest-synapse:buster"
+            build_arg: "SYTEST_IMAGE_TAG=buster"
           - sytest_image_tag: testing
             dockerfile: synapse
             tags: "matrixdotorg/sytest-synapse:testing"
+            build_arg: "SYTEST_IMAGE_TAG=testing"
+          - sytest_image_tag: bookworm
+            dockerfile: synapse
+            tags: "matrixdotorg/sytest-synapse:bookworm-python3.10"
+            build_arg: |
+              "SYTEST_IMAGE_TAG=bookworm"
+              "PYTHON_VERSION=python3.10"
```

Comment:
I think the `"` are redundant here, because the `|` means it's already a string from YAML's point of view.

(in fact, I'm a bit surprised they work. best get rid of them I think.)